test/test_compound_identifier.py

```diff
@@ -10,7 +10,7 @@ def test_is_input_header_positive(compound_identifier, header):
     """Test that valid input headers return True."""
     assert compound_identifier.is_input_header(header) is True
 
-@pytest.mark.parametrize("header", ["key", "inchiKey", "KEY", "INCHIKEY"])
```

Comment:
Hey @musasizivictoria please don't remove this test. You should add this test after your `input_header_negative` test in this PR.

Reply:
so sorry @DhanshreeA  , it was an oversight on method names

Reply:
Have resolved it i guess

Reply:
am ready to continue with other tests  once this is approved